README.md

```diff
@@ -1,60 +1,49 @@
-# generator-react-zeal
-
-Yeoman generator for Zeal's React boilerplate
-
-Expected to be used in the root of a Rails or Phoenix project configured with
-the Zeal client app strategy.
-
+### Yeoman Generator for [Zeal's](https://codingzeal.com/) React Boilerplate
 [![npm version](https://img.shields.io/npm/v/generator-react-zeal.svg)](https://www.npmjs.com/package/generator-react-zeal)
 [![CircleCI](https://circleci.com/gh/CodingZeal/generator-react-zeal.svg?style=shield&circle-token=62c69a799a2c2a4ad70148f0a4a398b38279ad91)](https://circleci.com/gh/CodingZeal/generator-react-zeal)
 
+This generator can be used with `yarn` or `npm`. `generator-react-zeal` is a [yeoman](http://yeoman.io/) generator and follows the usage pattern commonly found with the yo cli.
 
-## Installation
+***
```

Comment:
I like the separator, but I'd keep the `Installation` heading, then make the sub-headings `With Yarn` and `With npm`